content/docs/fields/select.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Select Field
+prev: /docs/fields/toggle
+next: /docs/fields/group
```

Comment:
Make sure to update `prev` and `next` for the toggle and the group docs as well

Reply:
oh righttttt